user-guide/Getting_started/The_DataMiner_User_Interface/Monitoring_app/Card_Pane/Element_Cards.md

```diff
@@ -18,7 +18,7 @@ The following pages are available:
 
   - ![Histogram icon](~/user-guide/images/MonitoringX_histogram2.png) : Displays a histogram for a trended table parameter.
 
-  - ![Trend graph icon](~/user-guide/images/MonitoringX_trend2.png) : From DataMiner 10.3.7/10.4.0 onwards: Allows you to access the [trending page](xref:Trending_Page).<!-- RN 36352 --> Prior to DataMiner 10.3.7/10.4.0: Allows you to view additional information on a trended parameter. When you click the icon, the parameter description, the parameter ID, and the time of the last change to the parameter are displayed. Below this, the *View trending* option provides access to the [trending page](xref:Trending_Page).
+  - ![Trend graph icon](~/user-guide/images/MonitoringX_trend2.png) : From DataMiner 10.3.7/10.4.0 onwards: Allows you to access the [trending page](xref:Trending_Page).<!-- RN 36352 --> Prior to DataMiner 10.3.7/10.4.0: Allows you to view additional information on a trended parameter. When you click the icon, the parameter description, the parameter ID, and the time of the last change to the parameter are displayed. Below this, the *View trending* option provides access to the [trending page](xref:Trending_Page). From DataMiner 10.0.0/10.0.2 onwards, a different trend icon can be displayed reflecting the recent trend behavior, see [Trend icons on data pages](xref:Working_with_behavioral_anomaly_detection#trend-icons-on-data-pages).
```

Comment:
Just to be sure, can the different trend icons also be displayed in the Monitoring app? I thought this was only in DataMiner Cube (and the section you've changed is related to the Monitoring app only).

Reply:
ah right, I didn't notice that this was about the Monitoring app. The pull request can be rejected.